core/spec/spec_helper.rb

```diff
@@ -6,7 +6,6 @@
 end
 
 require 'rspec/core'
-
```

Comment:
Not sure if this is intentional but I'd avoid this just to have a cleaner git history.